client/my-sites/hosting/main.js

```diff
@@ -65,6 +66,7 @@ class Hosting extends Component {
 			// Try to refresh the transfer state
 			this.props.fetchAutomatedTransferStatus( this.props.siteId );
 		}
+		setTimeout( () => scrollToAnchor( { offset: 30 } ), 2000 );
```

Comment:
It worked as is in General Settings, but in Hosting Configuration, it runs too early - SSH key data is not loaded yet, so the calculated height is incorrect.

Timeout fixes that, but it is hacky. Is there a way to check if child components finished loading? Or should we move SSH loading state up in the tree?

Reply:
It's hack y and it's not perfect.
It's a pity that this page has this dynamic content.
I tried adding two scrolls, one slow (1700ms) and another fast (300ms), but it has the same issues. It depends on the network speed to work fine.

I can see us creating a hook to detect if the element with anchor ID has relocated in the layout and moving the scroll automatically until the user scrolls manually.
Another option is to refactor the SSH keys card, replacing the spinner with some "placeholder bars loading" with the same height. Either way, we can revisit it later.
I've created this issue as follow up 1660-gh-Automattic/dotcom-forge

Reply:
Thanks for adding that ticket.